openmc/mgxs/library.py

```diff
@@ -4,6 +4,8 @@
 import pickle
 from numbers import Integral
 from collections import OrderedDict
+import numpy as np
```

Comment:
Per PEP8, imports should be ordered 1. standard library 2. third-party 3. local, with each group separated by a blank line